route/route.go

```diff
@@ -561,7 +459,7 @@ type batchedEvent struct {
 
 func (b *batchedEvent) getEventTime() time.Time {
 	if b.MsgPackTimestamp != nil {
-		return *b.MsgPackTimestamp
+		return b.MsgPackTimestamp.UTC()
```

Comment:
what are the implications of this?

Reply:
It prevents us from creating serialized times with the local server timezone, ie `2020-09-24T09:38:59.169594-07:00`. Functionally it shouldn't matter, especially since our servers should all be on UTC, but for local tests it keeps everything consistent.

Go's local-time-by-default is infuriating AF.